src/database/index.tsx

```diff
@@ -123,6 +119,68 @@ const buildUpdatePayload = (entity: any) => {
   return payload;
 };
 
+const timestampToDate = (timestamp: firebase.firestore.Timestamp): Date => {
+  return timestamp.toDate();
+};
+
+const buildPlannedRelease = (plannedReleaseData: any): PlannedRelease => {
```

Comment:
In the interest of keeping this file manageable, I might move these new functions out to a helper class in the package, e.g. `src/database/type-transforms.tsx` or `src/database/builders.tsx` or `src/database/date-handlers.tsx` or something.